homeassistant/components/watttime/sensor.py

```diff
@@ -77,13 +78,14 @@ def __init__(
     ) -> None:
         """Initialize the sensor."""
         super().__init__(coordinator)
-
-        self._attr_name = (
-            f"{description.name} ({entry.data[CONF_BALANCING_AUTHORITY_ABBREV]})"
-        )
```

Comment:
_attr_has_entity_name = True is set so it should stay the same

Reply:
It would turn them around, but potato potato